src/main/java/org/worshipsongs/importer/Song.java

```diff
@@ -0,0 +1,99 @@
+package org.worshipsongs.importer;
+
+/**
+ * Created by pitchumani on 10/8/15.
+ */
+public class Song
+{
+    String title, alternateTitle, author ,verseOrder ,songBook ,lyrics ,xmlLyrics ,searchTitle , searchLyrics;
+
+    public String getTitle()
+    {
+        return title;
+    }
+
+    public void setTitle(String title)
+    {
+        this.title = title;
+    }
+
+    public String getAlternateTitle()
+    {
+        return alternateTitle;
+    }
+
+    public void setAlternateTitle(String alternateTitle)
+    {
+        this.alternateTitle = alternateTitle;
+    }
+
+    public String getAuthor()
+    {
+        return author;
+    }
+
+    public void setAuthor(String author)
+    {
+        this.author = author;
+    }
+
+    public String getVerseOrder()
+    {
+        return verseOrder;
+    }
+
+    public void setVerseOrder(String verseOrder)
+    {
+        this.verseOrder = verseOrder;
+    }
+
+    public String getSongBook()
+    {
+        return songBook;
+    }
+
+    public void setSongBook(String songBook)
+    {
+        this.songBook = songBook;
+    }
+
+    public String getLyrics()
+    {
+        return lyrics;
+    }
+
+    public void setLyrics(String lyrics)
+    {
+        this.lyrics = lyrics;
+    }
+
+    public String getXmlLyrics()
+    {
+        return xmlLyrics;
+    }
+
+    public void setXmlLyrics(String xmlLyrics)
+    {
+        this.xmlLyrics = xmlLyrics;
+    }
+
+    public String getSearchTitle()
+    {
+        return searchTitle;
+    }
+
+    public void setSearchTitle(String searchTitle)
+    {
+        this.searchTitle = searchTitle;
+    }
+
+    public String getSearchLyrics()
+    {
+        return searchLyrics;
+    }
+
+    public void setSearchLyrics(String searchLyrics)
+    {
+        this.searchLyrics = searchLyrics;
+    }
```

Comment:
Override "equals", "hashCode" and "toString" methods with appropriate unit tests.